src/controllers/BanController.ts

```diff
@@ -13,14 +13,18 @@ const getBannedWord = async (): Promise<string> => {
     return bannedWordCache;
 };
 
-const ban = async (word: string): Promise<void> => {
+const getBanRequester = async(): Promise<string> => {
+    return (await db.get<BannedWordEntity>("settings", "banned-word")).banRequester;
```

Comment:
I chose to forgo caching here for two reasons:

  * I assume that most people actually won't call this function that much.
  * If we _want_ to move forward with caching, I think we should keep a cache of the actual database record that we retrieve, not just the banned word as a string (of course, this made sense when we only cared about the word and not who banned it).

Reply:
OH but I highly rec
```typescript
return (await db.get<BannedWordEntity>("settings", "banned-word"))?.banRequester;
```
The database could return `null`